### DISCUSSION TITLE
include file with uppercase letter

### DISCUSSION THREAD

**Discussion by angus-tsile** [bulletphysics/bullet3#3672] [General]
In "BulletCollision\BroadphaseCollision\btDbvt.cpp", there's an include file "LinearMath/btQuickProf.h".
This include file is btQuickprof.h with letter 'p' in lowercase, does not compile under Linux.

**Comment by erwincoumans** [comment]
Indeed, it is dead code (DBVT_ENABLE_BENCHMARK is never defined). Will fix anyway.
Thanks for the report!
